tests/aequilibrae/conftest.py

```diff
@@ -12,20 +12,64 @@
 from ..data import siouxfalls_project
 
 
+@pytest.fixture
+def create_project(create_project_session):
+    projects = []
+
+    def _create_project(name=None, source_dir=siouxfalls_project):
+        project = create_project_session(name=name, source_dir=source_dir, teardown=False)
+        projects.append(project)
+        return project
+
+    yield _create_project
+    for project in projects:
+        project.close()
+
+
+@pytest.fixture
+def create_empty_project(_empty_project, create_project):
+    def _create_empty_project(name=None):
+        return create_project(name=name, source_dir=_empty_project)
```

Comment:
create_project var shadows above method name.

Maybe for consistency: 

```suggestion
def create_empty_project(_empty_project, create_project_session):
    def _create_empty_project(name=None):
        return create_project_session(name=name, source_dir=_empty_project)
```

Reply:
urgh - i'm an idiot - never mind - that's HOW fixtures work.